<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Route Generator</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <layout class="QHBoxLayout" name="routeProperties">
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="windowTitle">
     <string>Route Generator v0.1</string>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_image"/>
    <addaction name="actionImport_Google_Map"/>
    <addaction name="actionSave_image"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionDraw_mode"/>
    <addaction name="actionPlayback"/>
    <addaction name="actionStop"/>
    <addaction name="actionGenerate_map"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Tutorial"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_image"/>
   <addaction name="actionImport_Google_Map"/>
   <addaction name="actionSave_image"/>
   <addaction name="separator"/>
   <addaction name="actionDraw_mode"/>
   <addaction name="actionNew_route"/>
   <addaction name="separator"/>
   <addaction name="actionPlayback"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionGenerate_map"/>
   <addaction name="separator"/>
   <addaction name="action_Quit"/>
  </widget>
  <action name="actionOpen_image">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/fileopen.png</normaloff>:/icons/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open image</string>
   </property>
   <property name="toolTip">
    <string>Open background image with map</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="actionSave_image">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/filesave.png</normaloff>:/icons/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save image</string>
   </property>
   <property name="toolTip">
    <string>Save the currently displayed image</string>
   </property>
  </action>
  <action name="actionDraw_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/desktop-32.png</normaloff>:/icons/icons/desktop-32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Draw mode</string>
   </property>
   <property name="toolTip">
    <string>(De)Activate route draw mode</string>
   </property>
  </action>
  <action name="actionGenerate_map">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/generate.png</normaloff>:/icons/icons/generate.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Generate route</string>
   </property>
   <property name="toolTip">
    <string>Generate the route into a selected directory</string>
   </property>
  </action>
  <action name="actionPlayback">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/player_play.png</normaloff>:/icons/icons/player_play.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Playback</string>
   </property>
   <property name="toolTip">
    <string>Playback the currently drawn route (preview)</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Tutorial">
   <property name="text">
    <string>&amp;Tutorial</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/player_stop.png</normaloff>:/icons/icons/player_stop.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;top</string>
   </property>
   <property name="toolTip">
    <string>Stops playback</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/undo.png</normaloff>:/icons/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/redo.png</normaloff>:/icons/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionImport_Google_Map">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/google-maps-logo.png</normaloff>:/icons/icons/google-maps-logo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import from Google Maps</string>
   </property>
   <property name="toolTip">
    <string>Import map from Google Maps</string>
   </property>
  </action>
  <action name="actionNew_route">
   <property name="icon">
    <iconset resource="routegen.qrc">
     <normaloff>:/icons/icons/new-route.png</normaloff>:/icons/icons/new-route.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Route</string>
   </property>
   <property name="toolTip">
    <string>Start New Route</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="routegen.qrc"/>
 </resources>
 <connections/>
</ui>
